macros/edr/system/system_utils/get_elementary_package_version.sql

```diff
@@ -0,0 +1,10 @@
+{# Only works if package is pulled from the registry. #}
+
+{% macro get_elementary_package_version() %}
+  {% for pkg in ref.config.packages.packages %}
```

Comment:
Is this the only way?

Reply:
Unfortunately yes. dbt doesn't provide a native way of doing so.